client/src/components/Graph/Graph.tsx

```diff
@@ -1120,18 +1125,21 @@ class Graph extends React.Component<GraphProps, GraphState> {
 
   updateColorTable(
     colors: RootState["colors"],
-    colorDf: Dataframe | null
+    colorDf: Dataframe | null,
+    isSpatial: boolean
   ): ColorTable {
     const { annoMatrix } = this.props;
     const { schema } = annoMatrix;
+
     /* update color table state */
     if (!colors || !colorDf) {
       return createColorTable(
         null, // default mode
         null,
         null,
         schema,
-        null
+        null,
+        isSpatial
```

Comment:
can I trouble you to take the opportunity to turn the params into an object, so we don't have to pass so many `null`s please 😁 🙏  ?

Thanks so much!